bundle/Resources/public/js/views/plugins/cof-copycontentplugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-copycontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the Copy Content plugin.
+     *
+     * @module cof-copycontentplugin
+     */
+    Y.namespace('cof.Plugin');
+
+    /**
+     * Provides plugin for the copy content plugin.
+     *
+     * @namespace cof.Plugin
+     * @class CopyContent
+     * @constructor
+     * @extends eZ.Plugin.CopyContent
+     */
+    Y.cof.Plugin.CopyContent = Y.Base.create('cofCopyContentPlugin', Y.eZ.Plugin.CopyContent, [], {
```

Comment:
Do you think it's a good idea to 'pollute' the UDW meta information (in PlatformUIBundle) by attributes needed by prototype bundle which everyone can disable in own project?